### PR TITLE
Some updates to the pod class

### PR DESCRIPTION
**Pull request by naved001**
The commit messages have more detail. The first commit updates `get_service_unit` and `write_metrics_by_pod`. The second commit does a clean up of things.

### CODE REVIEW

File: openshift_metrics/utils.py [hunk 1]
```diff
@@ -227,17 +188,11 @@ def write_metrics_by_namespace(condensed_metrics_dict, file_name, report_month):
 
 def write_metrics_by_pod(metrics_dict, file_name):
```
Comment (QuanMPhm) [inline openshift_metrics/utils.py:189]: A tiny nitpick, but I would have liked this renamed to `condensed_metrics_dict` for symmetry with `write_metrics_by_namespace()` and rest of the code.

**Review by QuanMPhm — COMMENTED** [review]
Just some small detail questions before I approve :)

File: openshift_metrics/invoice.py [hunk 2]
```diff
@@ -112,7 +114,46 @@ def get_service_unit(cpu_count, memory_count, gpu_count, gpu_type, gpu_resource)
 
     def get_runtime(self) -> Decimal:
         """Return runtime eligible for billing in hours"""
-        return Decimal(self.duration) / 3600
+        return Decimal(self.duration / 3600)
```
[the remark this answers is not in the document]
Comment (naved001) [reply openshift_metrics/invoice.py]: you are right, not sure why I made that change :/

File: openshift_metrics/utils.py (hunk 1 above)
Comment (naved001) [reply openshift_metrics/utils.py]: sure, I have changed that.

**Comment by naved001** [comment]
@QuanMPhm I have addressed your comments.

**Review by QuanMPhm — APPROVED** [review]

File: openshift_metrics/invoice.py [hunk 3]
```diff
@@ -150,13 +191,7 @@ class ProjectInvoce:
...
     def add_pod(self, pod: Pod) -> None:
         """Aggregate a pods data"""
-        su_type, su_count, _ = Pod.get_service_unit(
-            cpu_count=pod.cpu_request,
-            memory_count=pod.memory_request,
-            gpu_count=pod.gpu_request,
-            gpu_type=pod.gpu_type,
-            gpu_resource=pod.gpu_resource,
-        )
+        su_type, su_count, _ = pod.get_service_unit()
```
Comment (knikolla) [inline openshift_metrics/invoice.py:194]: Isn't `get_service_unit()` returning a `ServiceUnit` object now?

Comment (naved001) [reply openshift_metrics/invoice.py:194]: @knikolla It was returning a ServiceUnit namedtuple even before this PR. This doesn't change that. Is there anything wrong here?

**Review by knikolla — APPROVED** [review]

**Pull request merged**